.github/workflows/ci.yml

```diff
@@ -221,6 +221,7 @@ jobs:
         env:
           TEST_STATE_SCHEME: ${{ matrix.test-state-scheme }}
         run: |
+          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$GITHUB_WORKSPACE/target/lib
```

Comment:
Did we ever figure out if the Makefile is properly doing this for builds and this is working in the docker image? I'm fine with us exporting it here liberally just so that we know CI will always work, but I want to be sure that the UX doesn't get worse for the docker image or anyone working in the integration repo.

Reply:
we can delete this, I dont think this is needed anymore. This gets overwritten by espresso-run-tests-ci

Reply:
We can do it another PR, lets just merge this lol